.github/workflows/master.yml

```
@@ -7,6 +7,24 @@ on:
     branches: [ master ]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        INSTALL_LARGE_PYTHON_DEPS=true INSTALL_SMALL_PYTHON_DEPS=true source ./travis/install-common-deps.sh
+        pip install -r ./travis/lint-requirements.txt
```

```suggestion
      env:
        INSTALL_LARGE_PYTHON_DEPS: true
        INSTALL_SMALL_PYTHON_DEPS: true
      run: |
        ./travis/install-common-deps.sh
        pip install -r ./travis/lint-requirements.txt
```

We can use [the env context](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#env) here to set `INSTALL_LARGE_PYTHON_DEPS` and `INSTALL_SMALL_PYTHON_DEPS`.